### PR TITLE
opennbs: Add version 3.10.0

### LINKED ISSUE #10196: [Request]: OpenNoteBlockStudio
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

opennbs

### Description

An open-source Minecraft music maker.

### Homepage

https://github.com/OpenNBS/OpenNoteBlockStudio

### Download Link(s)

https://github.com/OpenNBS/OpenNoteBlockStudio/releases

### Some Indication of Popularity/Repute

GitHub stars: ~500

### PR DESCRIPTION
**Pull request by THEshezzee**
Closes #10196 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/opennbs.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "3.10.0",
+    "description": "An open-source Minecraft music maker.",
+    "homepage": "https://github.com/OpenNBS/OpenNoteBlockStudio",
```
Comment (HUMORCE) [inline bucket/opennbs.json:4]:
```suggestion
    "homepage": "https://opennbs.org/",
```

File: bucket/opennbs.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    },
+    "bin": "Minecraft Note Block Studio.exe",
+    "shortcuts": [
+        [
+            "Minecraft Note Block Studio.exe",
+            "Minecraft Note Block Studio"
+        ]
+    ],
+    "checkver": "github",
```
Comment (HUMORCE) [inline bucket/opennbs.json:19]:
```suggestion
    "checkver": {
        "github": "https://github.com/OpenNBS/OpenNoteBlockStudio"
    },
```

File: bucket/opennbs.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "description": "An open-source Minecraft music maker.",
+    "homepage": "https://github.com/OpenNBS/OpenNoteBlockStudio",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/OpenNBS/OpenNoteBlockStudio/releases/download/v3.10.0/Minecraft.Note.Block.Studio.zip",
+            "hash": "c6074646fa0774ca4a33ef258ca003428fd2916a5755f1e7c832f56950415350"
+        }
+    },
+    "bin": "Minecraft Note Block Studio.exe",
```
Comment (HUMORCE) [inline bucket/opennbs.json:12]:
```suggestion
```

File: bucket/opennbs.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/OpenNBS/OpenNoteBlockStudio/releases/download/v$version/Minecraft.Note.Block.Studio.zip"
+            }
+        }
+    }
+}
```
Comment (HUMORCE) [inline bucket/opennbs.json:27]:
```suggestion
}

```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

**Review by THEshezzee — COMMENTED** [review]
Done

**Comment by davidxuang** [comment]
Maybe [games](https://github.com/Calinou/scoop-games) is a better place, as you can only use this with the game.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**